shard.yml

```diff
@@ -6,4 +6,8 @@ authors:
 
 crystal: 0.35.1
 
+dependencies:
+  cable:
+    github: cable-cr/cable
```

Comment:
I specifically didn't include this shard as a dependency for a few reasons (which, admittedly, I should add to the docs):

- An application doesn't necessarily use Cable functionality even if it's using Turbo
- If an application _does_ use Cable, it has to configure itself for it explicitly so it isn't completely encapsulated
- I use [my own fork](https://github.com/jgaskins/cable) of that shard in my apps because I also use [a different Redis client](https://github.com/jgaskins/redis) 🙂

Reply:
@jgaskins okay, that explains it! :D Now it would fail though as it is trying to require something that won't exist.

I could change this dependency to yours if you would like that better. Your own fork of the cable looks nice. Couldn't you send some of those changes to cable-cr? That would be cool. If you have no time doing that I could do it and refer to you ofc.

Reply:
> Now it would fail though as it is trying to require something that won't exist.

Normally, I agree with this _so much_ and it's nice to hear people talking about dependencies like this!

I feel there's an important distinction here, though. In an application that uses Cable, Cable is exposed directly to application code via `Cable.configure` and your app's middleware stack. This means the application directly depends on Cable so it realistically wouldn't be left as a transitive dependency.

_However_, without at least including it as a development dependency, you're 100% right that it wouldn't compile to be able to test. Honestly, I haven't figured out how I'd even want to test this shard. I extracted most of it from an existing app which _also_ didn't have automated tests for Turbo-specific functionality. 😂 

> Your own fork of the cable looks nice. Couldn't you send some of those changes to cable-cr? That would be cool.

I agree! I would love to do that, actually. Even if it's just some of the cleanup. Maybe I can find some time over the next few days. If you're able to get to it before that, that's great, too! 🙂

Reply:
Okay good. I can remove the dependency for now and you can decide later when you know a good way to do it? :)